# Input: map position
geometry_msgs/Point position

---
# Output: normale della superficie
geometry_msgs/Vector3 normal
uint32 map_cell_x
uint32 map_cell_y
